narratives/ncov_sit-rep_ja_2020-03-13.md

```diff
@@ -95,18 +95,18 @@ COVID-19とそれを引き起こすウイルスであるSARS-CoV-2を理解す
 <!-- This is left-side text -->
 
 # [収集されたデータについて](https://nextstrain.org/ncov/2020-03-13?c=country&r=country&d=map&p=grid&legend=closed)
-現在、我々の元には、5大陸の30か国で採取されたサンプルの遺伝子配列があります。 これは信じられないほどの偉業です。今まさに流行が起きている状況下で、未知の大きな RNA ウイルスを分析することは難しいことです。これは世界中の科学者や医師による信じられないほどの作業とデータのタイムリーな共有によってのみ可能です。
+現在、私たちのもとには、5大陸の30か国で採取されたサンプルの遺伝子配列があります。 これは信じられないほどの偉業です。今まさに流行が起きている状況下で、未知の大きな RNA ウイルスを遺伝子配列解析をすることは難しいことです。これは世界中の科学者や医師による信じられないほどの作業とデータのタイムリーな共有によってのみ可能です。
 <br><br>このデータにより、多くの有用な特性を推測し、流行の広がりをリアルタイムで追跡することができますが、結論は利用可能なデータによって制限されることに留意して下さい。
-<br><br>たとえば、地図には南半球からの遺伝子配列がほとんど表示されていません。これは、COVID-19 がこれらの地域で流通していないため、またはこれらのケースを理解する重要性が無いためではありません。むしろ、これらの地域からの入手可能なデータが少ないことを意味します。マップ上のそれぞれの円のサイズは、流行の広がり度合いを示すものではなく、そのエリアにおいて利用可能なデータ量を示します。
+<br><br>たとえば、地図には南半球からの遺伝子配列がほとんど表示されていません。これは、COVID-19 がこれらの地域で伝搬してい _ない_ ため、またはこれらの症例を理解する重要性が _無い_ ためではありません。むしろ、これらの地域からの入手可能なデータが少ないことを意味します。マップ上のそれぞれの円のサイズは、実際の流行の広がり度合いを示すというよりも、そのエリアにおいて利用可能なデータ量を示します。
```

Comment:
First part is nits.

The latter part is hard to explain but...
If my understanding is correct, the sizes of circles represents **detected** (minimal?) sizes of spreads.
Because, in some sense, it shows the size of spreading, it may be better to remove the negative expression.

Reply:
Such nuances are important. Thank you for your advice.